pkg/envoy/lds/inmesh.go

```diff
@@ -50,6 +53,14 @@ func (lb *listenerBuilder) getInboundMeshFilterChains(proxyService service.MeshS
 			}
 			filterChains = append(filterChains, filterChainForPort)
 
+		case tcpAppProtocol:
+			filterChainForPort, err := lb.getInboundMeshTCPFilterChain(proxyService, port)
+			if err != nil {
+				log.Error().Err(err).Msgf("Error building inbound TCP filter chain for proxy:port %s:%d", proxyService, port)
+				continue // continue building filter chains for other ports on the service
```

Comment:
just because I'd like to hear what you had in mind, I imagine this is to at least not disrupt traffic for other applications if for some reason one inbound chain failed to generate?

Reply:
Yes, precisely.